src/utils/useLocationUpdateEffect.ts

```diff
@@ -1,8 +1,10 @@
 import { useEffect } from "react";
-import { UserInfo } from "firebase";
+import firebase, { UserInfo } from "firebase";
 
 import { updateUserProfile } from "pages/Account/helpers";
 
+const LOCATION_INCREMENT_SECONDS = 2;
```

Comment:
Wondering if we can write a little less often, not sure how much it'll load up Firebase.